src/tests/ftest/control/daos_admin_privileged.py

```diff
@@ -31,7 +31,10 @@ def test_daos_admin_format(self):
             Test daos_admin functionality to perform format privileged
             operations while daos_server is run as normal user.
 
-        :avocado: tags=all,pr,daily_regression,hw,small,daos_admin,basic
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,small
+        :avocado: tags=pmem
```

Comment:
Based on https://github.com/daos-stack/daos/blob/master/src/tests/ftest/control/daos_admin_privileged.yaml#L18-L20 this test should be using NVMe.  I wonder if it does not do any harm to add the tag now and then when the comment is removed in the yaml file it does not end up being forgotten.

Reply:
DAOS-4287, which is the reason why the nvme part is commented out in the yaml is already marked as resolved, I can try to uncomment the code and add the nvme tag